apps/balancer-tools/src/app/poolsimulator/(components)/SwapCurve.tsx

```diff
@@ -317,3 +364,60 @@ export function SwapCurve() {
     />
   );
 }
+
+function getBetaLimitIndexes({
+  analysisTokenOut,
+  analysisTokenIn,
+  tabTokenIn,
+  tabTokenOut,
+  analysisTokenInitialBalance,
+  tabTokenInitialBalance,
+  beta,
+}: {
+  analysisTokenOut: number[];
+  analysisTokenIn: number[];
+  tabTokenIn: number[];
+  tabTokenOut: number[];
+  analysisTokenInitialBalance: number;
+  tabTokenInitialBalance: number;
+  beta: number;
+}) {
+  const analysisTokenSwapAmounts = [...analysisTokenOut]
+    .reverse()
+    .concat(analysisTokenIn);
+
+  const tabTokenSwapAmounts = [...tabTokenIn].reverse().concat(tabTokenOut);
+
+  const analysisTokenBalanceAfterSwap = analysisTokenSwapAmounts.map(
+    (amount) => amount + analysisTokenInitialBalance,
+  );
+  const tabTokenBalanceAfterSwap = tabTokenSwapAmounts.map(
+    (amount) => amount + tabTokenInitialBalance,
+  );
+
+  const isBalancesInBetaRegion = analysisTokenBalanceAfterSwap.map(
+    (balanceX, index) => {
+      const balanceY = tabTokenBalanceAfterSwap[index];
+      return Math.abs(balanceX - balanceY) / balanceX + balanceY < beta;
```

Comment:
Shouldn't this be the following?
```suggestion
      return Math.abs(balanceX - balanceY) / (balanceX + balanceY) <= beta;
```
counterproof: `Math.abs(1 - 2) / 1 + 2 < .5` is false